client/js/store/modules/auth.js

```diff
@@ -1,8 +1,36 @@
-import { io } from '../../plugins/sails.js';
+import { io, reconnectSockets } from '../../plugins/sails.js';
 import { ROUTE_NAME_LOBBY, ROUTE_NAME_GAME } from '@/router';
 
 import { getPlayerPnumByUsername } from '_/utils/game-utils.js';
 
+async function handleLogin(context, username, password, signup = false) {
```

Comment:
This was the biggest lift, I had to rewrite all of the sign in/sign up logic to use fetch instead of the sockets, much like we do for the status check due to the way that browsers get and set cookies from the server.